GNUmakefile

```diff
@@ -19,6 +19,19 @@ fix-whitespace:
 check-whitespace:
 	cabal exec -- fix-whitespace --check
 
+# checking for files with long lines
+
+.PHONY : check-longfiles
+check-longfiles:
+# Limit is set to 100 characters. Due to a lot of files actually hitting that, currently this is
+# not a hard error, but simply a warning
+	@long_files=$$(rg -n --color always '.{101,}' -g "**/*.agda" -g "!**/Everything.agda") ; \
```

Comment:
To reduce output and only show offending filenames instead of all offending lines, could use `rg -l` here.